.gitignore

```diff
@@ -0,0 +1,5 @@
+/build/
+/.coverage
+/dist/
+/content_validator.egg-info/
```

Comment:
not sure if this implies we are building an egg, but we [should build a wheel](https://packaging.python.org/discussions/wheel-vs-egg/)?

Reply:
Fiddling with it on my machine to check what the various commands could create

Reply:
It is created also by `python setup.py bdist_wheel`, so it's just older naming for a modern dependency :+1: